modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -1818,8 +1836,27 @@ export class Wallet implements IWallet {
       throw error;
     }
 
+    let txPrebuildQuery;
```

Comment:
Is there a type we can assign here?

Reply:
`Promise<PrebuildTransactionResult | string>`